### PR TITLE
feat(ELY-943): Create binary auth action

### PR DESCRIPTION
**Pull request by akovtun-extenda**

### CODE REVIEW

File: binary-auth-attestation/src/create-sign-attestion.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const exec = require('@actions/exec');
+
+// Binauthz attestations gcloud cmd
+const createAttestation = async (
+  artifactUrl, attestor, attestorProject, keyversionProject, keyversionLocation,
+  keyversionKeyring, keyversionKey, keyversion) => {
+  const args = [
+    '--quiet',
+    'beta',
```
Comment (sasjo) [inline binary-auth-attestation/src/create-sign-attestion.js:9]: Do we need to use the beta command? Seems like `gcloud container binauthz` is available also.

File: binary-auth-attestation/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+### Examples
+
+```yaml
+jobs:
+  deploy-staging:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (sasjo) [inline binary-auth-attestation/README.md:19]: Use actions/checkout@v2 in docs.

File: binary-auth-attestation/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+      - uses: extenda/actions/setup-gcloud@v0
+        id: gcloud
```
Comment (sasjo) [inline binary-auth-attestation/README.md:22]: Remove `id` as we're not referencing it.

File: binary-auth-attestation/src/create-sign-attestion.js (hunk 1 above)
Comment (tomascarlqvist) [reply binary-auth-attestation/src/create-sign-attestion.js:9]: I can test that

**Review by sasjo — APPROVED** [review]
LGTM. Suggestions for changes on the docs primarily.

File: binary-auth-attestation/src/create-sign-attestion.js (hunk 1 above)
Comment (tomascarlqvist) [reply binary-auth-attestation/src/create-sign-attestion.js:9]: sign-and-create is not available for `gcloud container binauthz`

**Pull request merged**